src/containers/nextflow/nextflow.aws.sh

```diff
@@ -45,14 +45,15 @@ workDir = "$NF_WORKDIR"
 process.executor = "awsbatch"
 process.queue = "$NF_JOB_QUEUE"
 aws.batch.cliPath = "$AWS_CLI_PATH"
+env.LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/opt/aws-cli/bin"
```

Comment:
tested via #132 - this config option doesn't do what is needed.
can be reverted.